packages/utils/src/assets-utils/index.ts

```diff
@@ -0,0 +1,11 @@
+import { assets as assetList } from './assets';
+import { resolveIconPath } from './utils/resolveIconPath';
+
+export * from './constants';
+export * from './utils/network';
+export * from './utils/resolveIconPath';
+export * from './types';
+
+export const assets = assetList;
+
+export default resolveIconPath(process.env.BASE_URL || './', assetList);
```

Comment:
I usually dislike `export default` since it can result in inconsistent nomenclatures.

Reply:
Agreed, and I don't think we use it anywhere at all. I kept it there because for now I just copy pasted all the source from the old repo, but happy to make this change